examples/yoti_example_flask/app.py

```diff
@@ -2,16 +2,24 @@
 import os
 from os.path import join, dirname
 
+
 from dotenv import load_dotenv
 from flask import Flask, render_template, request
 
-from yoti_python_sdk import Client
+import sys
+
+sys.path.append(join(dirname(__file__), "../.."))
```

Comment:
Instead of just messing with the path, can we just `pip install` the SDK before running the example?  This means the latest version of the SDK will be included when you build the docker image, and we don't end up messing with the path

Reply:
If I understand... That means we wouldn't need this patchset at all? (Just some changes to the Dockerfiles?)